xml/transactional_updates.xml

```diff
@@ -0,0 +1,517 @@
+     to install additional packages, this will not work. Normal system
+     operations only modifying configuration files in <filename>/etc</filename>
+     will work.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     For <package>php7-fastcgi</package>, you must manually create a symlink,
+     <filename>/srv/www/cgi-bin/php</filename>, that points to
+     <filename>/usr/bin/php-cgi</filename>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <package>ntp</package>is part of the Legacy Module for migration from
+     older SLES versions. It is not supported on a new &productname;
+     installation, and has been replaced by <package>chrony</package>. If you
+     continue to use <package>ntp</package>, a fresh installation is required
+     to work correctly with transactional updates.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <package>sblim-sfcb</package>: The whole sblim ecosystem is incompatible
+     with transactional update.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <command>btrfs-defrag</command> from the
+     <package>btrfsmaintenance</package> package does not work with a read-only
+     root filesystem.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     For <command>btrfs-balance</command>, the variable
+     <literal>BTRFS_BALANCE_MOUNTPOINTS</literal> in
+     <filename>/etc/sysconfig/btrfsmaintenance</filename> must be changed from
+     <literal>/</literal> to <literal>/.snapshots</literal>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     For <command>btrfs-scrub</command>, the variable
+     <literal>BTRFS_SCRUB_MOUNTPOINTS</literal> in
+     <filename>/etc/sysconfig/btrfsmaintenance</filename> must be changed from
+     <literal>/</literal> to <literal>/.snapshots</literal>.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="sec.install-tu">
+  <title>Installing <package>transactional-update</package></title>
+
+  <para>
+   The <package>transactional-update</package> package is on &suse; Package
+   Hub, <link xlink:href="https://packagehub.suse.com/"/>. &suse; Package Hub
+   is available as a free extension to all users with a valid &productname;
+   subscription. You can enable the extension during installation, or
+   afterwards with &yast;, and then install the
+   <package>transactional-update</package> package like any other package with
+   Zypper or &yast;.
+  </para>
+
+  <para>
+   You may also elect to install
+   <package>transactional-update-zypp-config</package>, which prevents Zypper
+   from accidentally removing <package>transaction-update</package>.
+  </para>
+ </sect1>
+ <sect1>
+  <title>The <command>transactional-update</command> Command</title>
+
+  <important>
+   <title>Only use when requested by SUSE Support</title>
+   <para>
+    This reference for <command>transactional-update</command> should only be
+    used when requested by SUSE Support.
+   </para>
+  </important>
```

Comment:
No clue where this is coming from, but the customer has to use transactional-update of course for daily work on the system. It's the only way for the customer to install/remove/update packages.